frontend/src/apis/error/APIError.ts

```diff
@@ -0,0 +1,25 @@
+import { API_ERROR_MESSAGE } from '@/constants/messages/apiErrorMessage';
+
+const getErrorMessage = (errorCode: string): string => {
+  return API_ERROR_MESSAGE[errorCode as keyof typeof API_ERROR_MESSAGE] || '알 수 없는 에러가 발생했습니다.';
```

Comment:
```suggestion
  return API_ERROR_MESSAGE[errorCode as keyof typeof API_ERROR_MESSAGE] ?? '알 수 없는 에러가 발생했습니다.';
```
falsy를 막기위한 게 아니라 nullish를 막기 위한 것이니
null 병합연산자(`??`)를 쓰면 더 좋을거같네요

Reply:
좋은거 같습니다! 반영하겠습니다.